Userland/Games/Flood/BoardWidget.cpp

```diff
@@ -8,12 +8,30 @@
 
 #include <LibGUI/Menu.h>
 #include <LibGUI/Painter.h>
+#include <LibGfx/Palette.h>
 
-BoardWidget::BoardWidget(size_t rows, size_t columns, Vector<Color> colors, Color background_color)
+BoardWidget::BoardWidget(size_t rows, size_t columns)
     : m_board(make<Board>(rows, columns))
-    , m_background_color(background_color)
 {
-    m_board->set_color_scheme(move(colors));
+    update_color_scheme();
+}
+
+void BoardWidget::update_color_scheme()
+{
+    auto const& palette = GUI::Widget::palette();
+    Vector<Color> color_scheme {
+        palette.bright_black(),
+        palette.bright_red(),
+        palette.bright_green(),
+        palette.bright_yellow(),
+        palette.bright_blue(),
+        palette.bright_magenta(),
+        palette.bright_cyan(),
+        palette.bright_white()
+    };
```

Comment:
You can probably inline the construction of the vector into the `set_color_scheme` call